components/grid/templates/filter.md

```diff
@@ -305,6 +308,97 @@ For an example with the CheckboxList Filter, see the [Custom Data]({%slug grid-c
 
 ![Custom Filter Menu Template with Checkboxes](images/custom-filter-menu-checkboxes.png)
 
+## Filter Menu Buttons Template
+
+By default, the Filter Menu renders the `Filter` and `Clear` buttons. You can customize or remove them entirely by using the `FilterMenuButtonsTemplate` tag. 
+
+The template receives a `context` of type `FilterMenuTemplateContext` that provides the following members:
+
+* `FilterDescriptor` - the object that describes the column filter. By default it has two filters with the type and name of the field, and you can add more to its `FilterDescriptors` collection, or change its `LogicalOperator` from the default `AND`. You can use the filter descriptor to create a custom button that applies a predefined filter. 
```

Comment:
Replace the pronouns with the actual nouns to improve the readability:
- what does "it" refer to ("By default it has two filters...")?
- "its `FilterDescriptors`",  "its `LogicalOperator`"

```suggestion
* `FilterDescriptor`—the object that describes the column filter. By default, the column filter has two filters: one for the type and another for the name of the field. You can modify the column filter by:
   * Adding more filters to the `FilterDescriptors` collection.
   * Changing the `LogicalOperator` (`AND` by default).
   * Using the `FilterDescriptor` to create a custom button that applies a predefined filter. 
```